### ISSUE TITLE
Craft4?

### ISSUE THREAD

**Issue by gixxy22** [brambeekman/purge-assets#16]
Hi, any chance of getting this updated to support craft 4?

**Comment by brambeekman** [comment]
Yes! I'd start updating this today after two years of traveling the world!
